frontend/src/container/GuestUpload.jsx

```diff
@@ -75,13 +77,29 @@ class GuestUpload extends React.Component {
     }
   })
 
-  handleChangeFullName = e => this.setState({ guestName: e.target.value })
-  handleChangeComment = e => this.setState({ guestComment: e.target.value })
+  handleChangeFullName = e => this.setState({ guestFullname: { value: e.target.value, isInvalid: false } })
+  handleChangeComment = e => this.setState({ guestComment: e.target.value, isInvalid: false })
   handleChangePassword = e => this.setState({ guestPassword: { ...this.state.guestPassword, value: e.target.value } })
 
   handleAddFile = uploadFileList => {
+    const { props } = this
+
     if (!uploadFileList || !uploadFileList[0]) return
 
+    const alreadyUploadedList = uploadFileList.filter(file => this.state.uploadFileList.some(previousFile => previousFile.name === file.name))
```

Comment:
why the `.some()` has it's param named `previousFile` ? Is it related to `setState(previousState => {...})` ?

Reply:
Yes

Reply:
renamed